components/playlist/browser/resources/components/playlistsCatalog.tsx

```diff
@@ -0,0 +1,149 @@
+// Copyright (c) 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from 'react'
+import styled, { css } from 'styled-components'
+import { Link } from 'react-router-dom';
+
+import Icon from '@brave/leo/react/icon'
+import { color } from '@brave/leo/tokens/css'
+
+import * as PlaylistMojo from 'gen/brave/components/playlist/common/mojom/playlist.mojom.m.js'
+
+import PlayLaterCardOverlayImage from '../assets/playlater-card-overlay.svg'
+import PlaylistInfo from './playlistInfo'
+
+interface CatalogProps {
+  playlists: PlaylistMojo.Playlist[]
+}
+
+interface ThumbnailProps {
+  isDefaultPlaylist: boolean
+  thumbnailUrl: string | undefined
+}
+
+const StyledLink = styled(Link)<{}>`
+  text-decoration: none;
+  color:unset
+`
+
+const PlaylistContainer = styled.div<ThumbnailProps>`
+  /* Default Playlist background */
+  ${ p => p.isDefaultPlaylist && css`background: linear-gradient(150.64deg, #322FB4 0%, #3835CA 100%);`}
+
+  /* Default background for playlist without thumbnail */
+  ${ p => !p.isDefaultPlaylist && !p.thumbnailUrl && css`background-color: ${color.light.container.highlight};`}
+
+  /* Playlist with Thumbnail*/
+  ${ p => !p.isDefaultPlaylist && p.thumbnailUrl && css`
+      background: linear-gradient(0deg, rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)), url(${ p.thumbnailUrl });
+      background-size: cover;` }
+  border-radius: 8px;
+  padding: 8px;
+
+  position:relative;
+  display:flex;
+  gap: 16px;
+`
+
+const PlayLaterCardOverlay = styled.div<{}>`
+  position: absolute;
+  top: 0;
+  right: 0;
+  content: url(${PlayLaterCardOverlayImage});
+`
+
+const ColoredPlaylistInfo = styled(PlaylistInfo)<{ hasBackground: boolean }>`
+  color: ${ ({hasBackground}) => hasBackground ? color.dark.text.primary : color.light.text.primary };
+`
+
+const PlaylistThumbnailContainer = styled.div<ThumbnailProps>`
+  /* Default Playlist icon */
+  ${ p => p.isDefaultPlaylist && css`background-color: ${color.light.interaction.buttonPrimaryBackground}; color: white;` }
+
+  /* Default icon for playlist without thumbnail */
+  ${ p => !p.isDefaultPlaylist && !p.thumbnailUrl && css`background-color: ${color.light.container.background}; `}
```

Comment:
Same here, maybe the suggestion is wrong, but not sure:
```suggestion
  ${ p => !p.isDefaultPlaylist && !p.thumbnailUrl && css`background-color: ${color.container.background}; `}
```